xbmc/windowing/egl/EGLNativeTypeIMX.cpp

```diff
@@ -367,6 +367,19 @@ float CEGLNativeTypeIMX::GetMonitorSAR()
   if(!f_edid)
     return 0;
 
+  // first check if EDID is in binary format by reading 512bytes, compare 1st 8bytes
+  // against EDID 1.4 identificator [0x0,0xff,0xff,0xff,0xff,0xff,0xff,0x0]
+  // if no match, seek to 0 input file and continue with previous method.
+  if (((done = fread(&m_edid, 1, EDID_MAXSIZE, f_edid)) % 128) == 0 && done)
+    if (!memcmp(&m_edid, &EDID_HEADER, EDID_HEADERSIZE))
```

Comment:
address-of operation on reference returns same object so in that particular case &m_edid[0], m_edid and &m_edid is same value same type. 

I see the &x variant as clearer to understand when representing pointer. afaik gcc nor clang will not produce any hidden dereferencing so all should be equal at efficiency.

But if you wish the change I will do it.